xml2kinto/__main__.py

```diff
@@ -172,7 +194,9 @@ def main(args=None):
             xpath='certItems/*',
             kinto_client=kinto_client,
             bucket=args.cert_bucket,
-            collection=args.cert_collection),
+            collection=args.cert_collection,
+            schema=schemas.get(args.cert_bucket,
+                               schemas.get(CERT_COLLECTION))),
```

Comment:
You are right it should be the collection in both case. The rational was to use the defined collection arg first and fallback to the default.